core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionActor.scala

```diff
@@ -68,11 +74,11 @@ class WorkflowSubmissionActor(val dataSource: SlickDataSource,
 
     case Status.Failure(t) =>
       logger.error(t.getMessage)
-      scheduleNextWorkflowQuery
+      scheduleNextWorkflowQuery(pollInterval + (scala.util.Random.nextInt(1000) milliseconds))
```

Comment:
this makes me sad. i think `Future.succesful(ScheduleNextWorkflow)` might work too